### PR TITLE
Courier core update

### PR DESCRIPTION
**Pull request by alindgren**
This addresses issue #30 -- although it might be better to republish Courier.Core v 2.52.15.

### CODE REVIEW

File: src/Umbraco.Courier.Contrib.Resolvers/Umbraco.Courier.Contrib.Resolvers.csproj [hunk 1]
```diff
@@ -75,9 +75,9 @@
       <HintPath>..\packages\Examine.0.1.57.2941\lib\Examine.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="FluentNHibernate, Version=1.2.0.712, Culture=neutral, PublicKeyToken=8aa435e3cb308880, processorArchitecture=MSIL">
-      <HintPath>..\packages\Courier.Core.2.52.15\lib\FluentNHibernate.dll</HintPath>
-      <Private>False</Private>
+    <Reference Include="FluentNHibernate, Version=1.4.0.0, Culture=neutral, PublicKeyToken=8aa435e3cb308880, processorArchitecture=MSIL">
+      <HintPath>..\packages\Courier.Core.3.1.0\lib\FluentNHibernate.dll</HintPath>
+      <Private>True</Private>
```
Comment (leekelleher) [inline src/Umbraco.Courier.Contrib.Resolvers/Umbraco.Courier.Contrib.Resolvers.csproj:80]: I'd previously sent a PR to set "Copy Local" to false, so the dependency DLLs aren't copied over to the output /bin folder.
I think NuGet updates automatically set it back to "Copy Local" when re-referencing.

However the build script does only grab the "Umbraco.Courier.Contrib.Resolvers.dll" file, so it's not a show-stopper.
Guess I was looking for a _cleaner_ build. 😃

**Comment by Shazwazza** [comment]
I had to do this too... it's not perfect because it means that there's a small chance that this contrib proj won't be backwards compat with previous courier versions if 'new' apis are used... but i think that would be very rare and in any case it's in everyone's best interest to use the latest Courier version.

It would be 'easier' to republish the original myget nuget package, problem is how the heck can i figure out how to get it back! :) I'll see what i can do

**Comment by leekelleher** [comment]
@Shazwazza - I found a copy of "Courier.Core.2.52.15.nupkg" in my local NuGet cache.

http://code.leekelleher.com/umbraco/archive/courier/Courier.Core.2.52.15.nupkg

**Comment by Shazwazza** [comment]
Thank you sir!! i'll see if i can get it uploaded

**Comment by leekelleher** [comment]
Also found http://code.leekelleher.com/umbraco/archive/courier/UaaS.Courier.UI.2.52.15.nupkg
If that's helpful too?

**Comment by Shazwazza** [comment]
Thanks @leekelleher  that's uploaded now, I think we can resolve these issues now

**Comment by Shazwazza** [comment]
don't think the UI one is needed for the contrib proj

**Pull request closed without merging**